backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/SyncToRemoteStatus.java

```diff
@@ -5,7 +5,7 @@
 @Data(staticConstructor = "of")
 public class SyncToRemoteStatus {
   public enum Relation {
-    Untracked, Ahead, Behind, DivergedAndNewerThanRemote, DivergedAndOlderThanRemote, InSync
+    Untracked, Ahead, Behind, DivergedAndNewer, DivergedAndOlder, InSync
```

Comment:
But... but... as we discussed... this is going to be confusing soon... what is ahead of what... what is newer than what... :frowning_face:

Reply:
I'm in favor of `{Of,Than,With,}Remote` everywheeeere

Reply:
The class name is `SyncToRemoteStatus` so relations reference status of a branch to its remote. Isn't it clear?

Reply:
Hmmm right... ok, let's leave as is as for now, that's been renamed too many times ;D

Reply:
The **final** version:
- Untracked
- AheadOfRemote
- BehindRemote
- DivergedFromAndNewerThanRemote
- DivergedFromAndOlderThanRemote
- InSyncWithRemote